app/helpers/followers_helper.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module FollowersHelper
+
+  # render a follow or unfollow btn for specific user
+  def followers_btn(user)
+    if user_signed_in? && current_user.id != user.id
+      @@following = user.followed_by(current_user.id)
+      if @@following.present?
+        unfollow_btn
+      else
+        follow_btn(user)
+      end
+    end
+  end
+
+  private
+
+  def follow_btn(user_to_follow)
+    form_with model: Follower, method: :post do |form|
+      res = form.hidden_field :followed_user_id, value: user_to_follow.id
+      res += form.hidden_field :followed_by_user_id, value: current_user.id
+      res += form.button '', class: 'btn badge bg-primary rounded-pill', title: 'Follow' do
+        'Follow <i data-feather="user-plus"></i>'.html_safe
+      end
+      res
+    end
+  end
```

Comment:
@jshidalgokommit Why are we putting frontend logic into a helper? If you need to do `DRY` (Don't repeat yourself), you can create a `_form` partial file. What do you think?

Reply:
Ok, helpers should not be used like this